modelに追記

mamiko-7172 opened the pull request:
以下追記しましたのでご確認ください

〇Category.php
←store,reviewとの連携
〇Favorit.php
←store,userとの連携
〇Prefecture.php
←storeとの連携
〇Review.php
←store,user,categoryとの連携
〇Store.php
←reveiw,favorite,category,prefectureとの連携
〇User.php
←review,favoriteとの連携

mamiko-7172 committed:
modelに追記

以下追記しましたのでご確認ください

〇Category.php
←store,reviewとの連携
〇Favorit.php
←store,userとの連携
〇Prefecture.php
←storeとの連携
〇Review.php
←store,user,categoryとの連携
〇Store.php
←reveiw,favorite,category,prefectureとの連携
〇User.php
←review,favoriteとの連携

tai-namechan reviewed (approved):
いいと思います！
LGTM!!

tai-namechan reviewed (changes requested):
user.phpの40行目reviewメソッドなんですけど、reviewsじゃないとテーブルの値が受け取れません
approve後にコメントしちゃってあれなんすけど、修正お願いします！

tai-namechan commented on app/User.php:
```diff
@@ -36,4 +36,14 @@ class User extends Authenticatable
         'email_verified_at' => 'datetime',
     ];
+
+    function review()
```
ここです

mamiko-7172 committed:
Modelの追記に関して

mamiko-7172 reviewed (commented):
@tai-namechan 
ご確認ありがとうございます。
メソッドが単数形になっていてすいません。
全部複数形に直しました！！

ArisaNAKAMURA-GitHub commented on app/Favorite.php:
```diff
@@ -6,5 +6,13 @@
+        return $this->belongsTo('App\Store');
+    }
+
+    function users()
```
［MUST］userでいいと思います〇

ArisaNAKAMURA-GitHub commented on app/Review.php:
```diff
@@ -6,6 +6,18 @@
 {
-    //
+    function stores()
+    {
```
［MUST］storeでいいと思います〇

ArisaNAKAMURA-GitHub commented on app/Review.php:
```diff
@@ -6,6 +6,18 @@
+    }
     
+    function users()
+    {
```
［MUST］userでいいと思います〇

ArisaNAKAMURA-GitHub commented on app/Review.php:
```diff
@@ -6,6 +6,18 @@
+        return $this->belongsTo('App\User');
+    }
+
+    function categories()
```
［MUST］categoryでいいと思います〇

ArisaNAKAMURA-GitHub commented on app/Store.php:
```diff
@@ -6,5 +6,23 @@
+        return $this->hasMany('App\Favorites');
+    }
+
+    function categories()
```
［MUST］categoryでいいと思います〇

ArisaNAKAMURA-GitHub commented on app/Store.php:
```diff
@@ -6,5 +6,23 @@
+        return $this->belongsTo('App\Category');
+    }
+
+    function prefectures()
```
［MUST］prefectureでいいと思います〇

ArisaNAKAMURA-GitHub commented on app/Favorite.php:
```diff
@@ -6,5 +6,13 @@
 class Favorite extends Model
 {
-    //
+    function stores()
```
［MUST］一つのお気に入りは一つのお店にできるものなので、storeでいいと思います〇

ArisaNAKAMURA-GitHub reviewed (changes requested):
ごめんなさい。私も追加で変更をお願いしたいです。
一対多の関係のところばかりだったはずなので、メソッド名は全部を複数にしなくていいと思います。

mamiko-7172 committed:
複数形に関しての修正しました

mamiko-7172 commented:
@ArisaNAKAMURA-GitHub 
確認いただきありがとうございます。
複数形と単数形の理解をしてなくてすいません^^;
コメントいただいた点修正しました。

tai-namechan reviewed (approved):
確認しました！修正お疲れ様です

mamiko-7172 committed:
indexページに関して追記しました

〇StoreController
→・５～7行目にuseを3つ追記
　・indexメソッドにデータを取ってくるコードと記入

〇index.blade.php
→・データを引っ張る項目のコードを変更

ArisaNAKAMURA-GitHub reviewed (approved):
![b6b61869a7c77523f38b5053865a243a](https://user-images.githubusercontent.com/73116350/120800452-edb9f080-c57a-11eb-90ae-631650abd5e9.png)
データ引っ張ってくるのもいけると思います！確認するためにもダミーデータが欲しいところですねこれ…

Mkk818 commented on app/Http/Controllers/StoreController.php:
```diff
@@ -13,7 +16,9 @@ class StoreController extends Controller
     {
-        return view('index');
+        $stores = Store::all();
+        $comments = Review::find(1)->reveiws()->where('comment')->first();
```
[IMO]
この部分の20行目のfind(1)->のあとはreviews()ではないでしょうか？
もしこれで正解だったらすみません…
